_tour/implicit-conversions.md

```diff
@@ -38,6 +38,45 @@ In the second case, a conversion `c` is searched for, which is applicable to `e`
 
 An example is to compare two strings `"foo" < "bar"`. In this case, `String` has no member `<`, so the implicit conversion `Predef.augmentString("foo") < "bar"` is inserted. (`scala.Predef` is automatically imported into all Scala programs.)
 
+### How are implicit conversions brought into scope? ###
+
+{% tabs implicit-conversion-scope class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+In Scala 2, an implicit convesion is brought into scope by importing from `ImplicitConversions`.
+
+```scala mdoc
+case class Student(name: String) {
+  def printName: Unit = println(name)
+}
+  
+object Conversions {
+  implicit def fromStringToStudent(name: String): Student = Student(name)
+}
+
+import ImplicitConversions._
+object Usage:
+  def main(args: Array[String]) = "Reginald".printName
+```
+{% endtab %}
+{% tab 'Scala 3' %}
+In Scala 3, an implicit conversion must be explicitly imported in order to be applicable.
```

Comment:
this isn't true, you can also have a given import:
```scala
import Conversions.given
import Conversions.{given Conversion[String, Student]}
```

Reply:
or if the implicit conversion is in the companion object of the argument type, (e.g. String in this case) then no import is necessary

Reply:
Thank you for all the suggestions! I've learned a lot! I've also improved the documentation.

Reply:
Just asking: There's no way to add a companion object to a Java class like `java.lang.String`, is it? I tried that but either the compiler runs into trouble or it does not find the conversion.

Reply:
companion objects must be defined in the same source file :)